op-e2e/actions/l2_verifier.go

```diff
@@ -121,6 +121,10 @@ func (s *l2VerifierBackend) StopSequencer(ctx context.Context) (common.Hash, err
 	return common.Hash{}, errors.New("stopping the L2Verifier sequencer is not supported")
 }
 
+func (s *l2VerifierBackend) SequencerStopped(ctx context.Context) bool {
+	return true
```

Comment:
why return true？

Reply:
This module is for integration tests only. And `l2VerifierBackend.SequencerStopped()` will never be used. I just implement it because of Go interface requirement.